core/src/test/java/org/everit/json/schema/loader/DefinesPropertyTest.java

```diff
@@ -113,7 +114,7 @@ public void testOfTest() {
 
     @Test
     public void patternPropsAndSchemaDefs() {
-        ObjectSchema actual = (ObjectSchema) SchemaLoader.load(get("patternPropsAndSchemaDeps"));
+        ObjectSchema actual = (ObjectSchema) loadAsV201909(get("patternPropsAndSchemaDeps"));
```

Comment:
Please try to avoid changing existing tests, add a new one instead (same comment for the above tests in the class and in `testschemas.json`).

Reply:
My understanding was that the purpose of these tests to test `definesProperty` method and the emphasis is not on the loading which is tested separately in ObjectSchemaLoaderTest.java. That is why I finally did not duplicate the tests just thought to update the loading to the latest specification.